pkg/kubelet/apis/cri/v1alpha1/runtime/constants.go

```diff
@@ -25,3 +25,30 @@ const (
 	// NetworkReady means the runtime network is up and ready to accept containers which require network.
 	NetworkReady = "NetworkReady"
 )
+
+// LogStreamType is the type of the stream in CRI container log.
+type LogStreamType string
+
+const (
+	// Stdout is the stream type for stdout.
+	Stdout LogStreamType = "stdout"
+	// Stderr is the stream type for stderr.
+	Stderr LogStreamType = "stderr"
+)
+
+// LogTag is the tag of a log line in CRI container log.
+// Currently defined log tags:
+// * First tag: Partial/End - P/E.
+// We can define new tags in the future if needed. However, if we find out
```

Comment:
nit:
```
The field in the container log format can be extended to include multiple
tags by using a delimiter, but changes should be rare. If it becomes clear
that better extensibility is desired, a more extensible format (e.g., json)
should be adopted as a replacement and/or addition.
```
or just not write things about the future in the api  :-)

Reply:
Done.